### ISSUE TITLE
make-php - Make output file content pretty

### ISSUE THREAD

**Issue by sovetski** [wp-cli/i18n-command#395]
## Feature Request

- [x] Yes, I reviewed the [contribution guidelines](https://make.wordpress.org/cli/handbook/contributing/).

**Describe your use case and the problem you are facing**

When we do `wp i18n make-php` it creates PHP files containing one line, like minified PHP, but If we want to delete the old .po/.mo files to work with only PHP translations, working one line generated files is not practical

![image](https://github.com/wp-cli/i18n-command/assets/13520683/a933ee0b-3f56-4387-bd66-f9fc960d4933)

**Describe the solution you'd like**

Formatted output will look better

![image](https://github.com/wp-cli/i18n-command/assets/13520683/4f5318dc-6ec8-4a22-bc17-f6d8ba5c5051)

**Comment by swissspidy** [comment]
We can add a `pretty-print` option like we have for make-json. Wanna help put a PR together? :)

**Comment by sovetski** [comment]
> We can add a `pretty-print` option like we have for make-json. Wanna help put a PR together? :)

Done: https://github.com/wp-cli/i18n-command/pull/396

Let me know if something is not OK 😊

**Comment by sovetski** [comment]
@kkmuffme yes exactly it is to not use .po files anymore, php files are more readable and easy to edit
